connection.js

```diff
@@ -1583,6 +1583,12 @@ class Connection {
             return;
         }
 
+        // Warn if we hit the maximum parsed header lines limit
+        const max_lines = trans.MAX_HEADER_LINES;
+        if (this.transaction.header_lines.length >= max_lines) {
```

Comment:
What's the utility of creating max_lines?  You can now:

`if (this.transaction.header_lines.length >= trans.MAX_HEADER_LINES) {`

right?

Reply:
Makes sense since we're not using it more than once